parent/pom.xml

```diff
@@ -166,7 +166,7 @@
     <commons-text-version>1.2</commons-text-version>
     <commons-vfs2-version>2.0</commons-vfs2-version>
     <compress-lzf-version>1.0.4</compress-lzf-version>
-    <consul-client-version>0.16.3</consul-client-version>
+    <consul-client-version>1.0.0</consul-client-version>
     <consul-client-bundle-version>0.16.2_1</consul-client-bundle-version>
```

Comment:
You need to change the bundle version too and check if the Karaf feature still work. Thanks.